RNFSManager.m

```diff
@@ -323,7 +323,7 @@ + (BOOL)requiresMainQueueSetup
     }
 
     // Seek to the position if there is one.
-    [file seekToFileOffset: (int)position];
```

Comment:
It seems like we can just remove the `*` from line 296 and 297 then remove the casting here and line 330 and we'll be ok